api/src/main/resources/client_api.yaml

```diff
@@ -74,6 +74,57 @@ paths:
           schema:
             $ref: "#/definitions/MaterializeCohortResponse"
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/concepts:
+      get:
+        security: []
+        tags:
+          - concepts
+
+        description: >
+          Searches for concepts in concept table by name, and optionally filter on
+          domain ID, vocabulary ID, or standard concept status. Uses the CDR version affiliated
+          with the workspace specified in the path.
+
+        operationId: "searchConcepts"
+        parameters:
+          - in: query
+            name: query
+            type: string
+            required: true
+            description: >
+              A query string that can be used to match a subset of the name (case-insensitively),
+              the entire code value (case-insensitively), or the concept ID.
+          - in: query
+            name: standard_concept
+            type: boolean
+            required: false
+            description: >
+              True if only standard concepts should be returned, false if any concepts
+              can be returned.
+          - in: query
+            name: vocabulary_id
+            type: string
+            required: false
+            description: The vocabulary ID for the concepts returned (e.g. SNOMED, RxNorm)
+          - in: query
+            name: domain_id
+            type: string
+            required: false
+            description: >
+             The domain ID for the concepts returned (e.g. Observation, Drug). TODO: consider
```

Comment:
Is the full list documented somewhere? Here and above

Reply:
Not right now. I plan to have constants in the client library for this in the short term; long term I was going to use an enum here.